sdk/tests/tutorials/properties/test_derived_properties.py

```diff
@@ -56,7 +56,8 @@ def create_ratings_property(self, *ratings):
                         f"Property {property_definition.domain}/{property_definition.scope}/{property_definition.display_name} already exists"
                     )
             finally:
-                self.id_generator.add_scope_and_code("property_definition", property_definition.scope, property_definition.code, ["Instrument"])
+                self.id_generator.add_scope_and_code("property_definition", property_definition.scope,
+                                                     property_definition.code, ["Instrument"])
```

Comment:
is there some rogue formatting happening here?

Reply:
yeah, turned on format on save.  It made a few changes in some places.

Reply:
fixed